src/en/reference-install.md

```diff
@@ -167,9 +167,9 @@ instructions on how to build Juju from source.
 [snapcraft]: https://snapcraft.io
 [snapd-install]: https://snapcraft.io/docs/core/install
 [juju-new-bug]: https://bugs.launchpad.net/juju/+filebug
-[juju-win-signed]: https://launchpad.net/juju/2.3/2.3.7/+download/juju-setup-2.3.7-signed.exe
-[juju-win-signed-md5]: https://launchpad.net/juju/2.3/2.3.7/+download/juju-setup-2.3.7-signed.exe/+md5
-[juju-centos]: https://launchpad.net/juju/2.3/2.3.7/+download/juju-2.3.7-centos7.tar.gz
-[juju-centos-md5]: https://launchpad.net/juju/2.3/2.3.7/+download/juju-2.3.7-centos7.tar.gz/+md5
+[juju-win-signed]: https://launchpad.net/juju/2.3/2.3.8/+download/juju-setup-2.3.8-signed.exe
```

Comment:
looks like the signed Windows binary isn't there yet (I can only find https://launchpad.net/juju/2.3/2.3.8/+download/juju-setup-2.3.8.exe)

Reply:
I know, I put a note in my opening comment. :unicorn:

Reply:
Sorry!